README.md

```diff
@@ -63,29 +37,20 @@ After you're finished please remove all the comments and instructions!
 
 ### Tech Stack <a name="tech-stack"></a>
 
-> React.
-
-<!-- Features -->
+> React. 
 
 ### Key Features <a name="key-features"></a>
 - **[Display a list of books ..]**
 - **[Allows users to Add a book ..]**
 - **[Allows users to Remove a selected book..]**
 
-<p align="right">(<a href="#readme-top">back to top</a>)</p><!-- LIVE DEMO -->
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-- [Live Demo Link](https://emekaemmanuel.github.io/Bookstore
-) 
-
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-## Project Presentation <a name="project-presentation"></a>
-
-- [Project Presentation video link](https://) 
-
-<!-- GETTING STARTED -->
+## Project Presentation <a name="project-presentation"></a> 
```

Comment:
As the previous reviewer requested, kindly remove this section heading as well. Sections with only heading and no info look less professional 👍